### PR TITLE
Revert globalization of Swagger descriptions [2.x]

### PR DESCRIPTION
**Pull request by bajtos**
As discussed in SlackChat, Swagger descriptions should not be globalized. Translating Swagger is an open problem, see e.g. https://github.com/OAI/OpenAPI-Specification/issues/274

@0candy please review

### CODE REVIEW

**Comment by 0candy** [comment]
LGTM.

File: lib/model.js [hunk 1]
```diff
@@ -7,12 +7,11 @@
  * Module Dependencies.
  */
 
-var g = require('strong-globalize')();
```
Comment (0candy) [inline lib/model.js]: You can't remove this line. Other places in this file are using it and failing CI linter.

Comment (bajtos) [inline lib/model.js]: Oh of course, this was not intentional. Will fix.

**Comment by nacho4d** [comment]
This was exactly what I was looking for. OAI/OpenAPI-Specification#274 was closed (by a missunderstanding I think)

**Pull request merged**